dts/bindings/stepper/adi/adi,tmc50xx.yaml

```diff
@@ -85,7 +85,7 @@ properties:
 
 child-binding:
   include:
-    - name: stepper-controller.yaml
+    - name: stepper.yaml
```

Comment:
i think only `en-gpios` and `invert-direction` should be allowed for this driver. Looking into the specs i dont see other features available over gpio (microstep config, step/dir etc)

Reply:
oh ok. probably the model i checked was an exception and did not have those pins exposed

Reply:
rsolved